File: 1551-minimum-operations-to-make-array-equal/1551-minimum-operations-to-make-array-equal.py
class Solution:
    def minOperations(self, n: int) -> int:
        m = n//2
        ans = 0
        if n % 2:
            i = 2
            while m:
                ans += i
                i += 2
                m -= 1

        else:
            i = 1
            while m:
                ans += i
                i += 2
                m -= 1

        return ans


        

File: 1551-minimum-operations-to-make-array-equal/1551-minimum-operations-to-make-array-equal.js
/**
 * @param {number} n
 * @return {number}
 */
var minOperations = function(n) {
    let m = Math.floor(n / 2);
    let ans = 0;
    
    if (n % 2 !== 0) {
        let i = 2;
        while (m) {
            ans += i;
            i += 2;
            m -= 1;
        }
    } else {
        let i = 1;
        while (m) {
            ans += i;
            i += 2;
            m -= 1;
        }
    }
    return ans;
};